### DISCUSSION TITLE
Remove Top Header

### DISCUSSION THREAD

**Discussion by jaydenrussell** [wazuh/wazuh#16767] [Ideas]
The top header is 50 pixels high just to have the login info at the right side. If the GlobalHeaderBars could be consolidated with the Logo and menu on the left and login status on the right, that would clear up some screen real estate.

**Comment by s-ocando** [comment]
Hi @jaydenrussell, 

I'm happy to let you know that with our most recent release (4.4.x) this issue has been addressed.
Here's a comparison of the interface for freshly installed Wazuh v4.3.10 and Wazuh v4.4.1: 

4.3.10             |  4.4.1
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61882981/233558820-3849f1a3-2cf0-45c1-838a-d217ee572734.png)  |  ![image](https://user-images.githubusercontent.com/61882981/233558988-4fd49909-159c-4f73-b0e7-8aba6c7fcf0f.png)

Let us know if you have any questions.
